Vagrantfile

```diff
@@ -171,12 +174,24 @@ def post_up_message
   msg = 'Your Trellis Vagrant box is ready to use!'
   msg << "\n* Composer and WP-CLI commands need to be run on the virtual machine."
   msg << "\n* You can SSH into the machine with `vagrant ssh`."
-  msg << "\n* Then navigate to your WordPress sites at `/srv/www`"
-  msg << "\n  or to your Trellis files at `#{ANSIBLE_PATH_ON_VM}`."
-
+  msg << "\n* Your WordPress sites are found in `/srv/www`."
+  msg << "\n* Windows users: Trellis files are in `#{ANSIBLE_PATH_ON_VM}`." if Vagrant::Util::Platform.windows?
   msg
 end
 
 def remote_site_path(site_name, site)
   "/srv/www/#{site_name}/#{site['current_path'] || 'current'}"
 end
+
+def which(cmd)
+  exts = ENV['PATHEXT'] ? ENV['PATHEXT'].split(';') : ['']
+
+  ENV['PATH'].split(File::PATH_SEPARATOR).each do |path|
+    exts.each do |ext|
+      exe = File.join(path, "#{cmd}#{ext}")
+      return exe if File.executable?(exe) && !File.directory?(exe)
+    end
+  end
+
+  nil
+end
```

Comment:
Here's an issue that you may figure should be addressed by the user, not by Trellis.

I use pyenv to manage python versions on my machine. Even after uninstalling Ansible, a shim remains at `/usr/local/var/pyenv/shims/ansible-playbook`, causing an incorrect return value from this `which(cmd)`.

Here's a potential change that resolves it on my machine (dev/null handling from [here](http://stackoverflow.com/a/35142176/6847025)):
```diff
- return exe if File.executable?(exe) && !File.directory?(exe)
+ return system("#{exe} --help", [:out, :err] => File::NULL) if File.executable?(exe) && !File.directory?(exe)
```
This appeals to me because it tests whether the `ansible-playbook` command actually works rather than just whether it should work, the latter being assumed based on the presence of the `exe`. I don't know whether this revised approach would be good practice or acceptable ruby.

It the change above is applied, it returns `true`, `false`, or `nil`, if I recall correctly (instead of `exe` path). So, maybe the name `which()` should change to something like `cmd_valid(cmd)`.